index.html

```diff
@@ -1,6 +1,57 @@
-<head>
-  <script type="text/javascript" src="https://firefoxux.github.io/SharedBanner/fxux-banner.min.js"></script>
-</head>
-<body>
- <h1>Hello there!</h1>
-</body>
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Firefox Design</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://code.cdn.mozilla.net/fonts/fira.css">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <style>
+      :root {
+        --blue-50: #0a84ff;
+        --blue-60: #0060df;
+        --blue-70: #003eaa;
+        --blue-80: #002275;
+      }
+
+      .fira { font-family: "Fira Sans"; }
+
+      .font-smoothing {
+        -moz-osx-font-smoothing: grayscale;
+        -webkit-font-smoothing: antialiased;
+      }
+    </style>
+  </head>
+  <body class="fira font-smoothing">
+    <a href="/photon">
+      <div class="white vh-25 pa4" style="background-color: var(--blue-50)">
+        <p class="ma0 fw5 f4 f2-m f1-l dib mr2 mr3-ns">Photon Design System</p>
+        <svg fill="white" xmlns="http://www.w3.org/2000/svg" class="h1 w1 h2-ns w2-ns" viewBox="0 0 12 12"><path d="M11.92.62A1,1,0,0,0,11,0H6A1,1,0,0,0,6,2H8.59L4.79,5.79A1,1,0,1,0,6.21,7.21L10,3.41V6a1,1,0,0,0,2,0V1A1,1,0,0,0,11.92.62Z"/><path d="M8,12H3A3,3,0,0,1,0,9V4A1,1,0,0,1,2,4V9a1,1,0,0,0,1,1H8a1,1,0,0,1,0,2Z"/>
```

Comment:
It looks like we're using the same SVG in all of these places.  Let's pull it out into its own file, so that I can look at it in my browser and see what it looks like.  (Actually, even if we aren't using the same svg, let's pull them out into their own files. 😉)